### ISSUE TITLE
Fails to submit for approval without comment

### ISSUE THREAD

**Issue by callumbwhyte** [nathanwoulfe/Plumber-2#11]
When requesting approval to publish a content item, if you don't supply a comment the task doesn't get correctly saved. The comment field is not required in the UI.

Interestingly the task is still partially recorded as it shows in the "pending" count in the dashboard but isn't visible anywhere else...

Looks like a simple schema change is needed:

```
Could not initiate workflow: "Cannot insert the value NULL into column 'AuthorComment', table 'dbo.WorkflowInstance'; column does not allow nulls. INSERT fails. The statement has been terminated."
```

This is on Plumber v1.0.7 and Umbraco 8.4

**Comment by nathanwoulfe** [comment]
Hey mate, you're not the first to raise this, but I can't replicate it - I can never submit for approval without entering a comment, the button is always disabled. 

Must be something changing in the packaged version compared to running from source.

**Comment by nathanwoulfe** [comment]
So. I run grunt-angular-template-embedding as part of the build, which replaces templateUrl references in components with the inlined template - reduces requests, less files shipped etc. For some reason, it's removing attributes off particular components, namely the umbButton and umbToggle directives, which means they never compile correctly so the state isn't set.

Hopefully it's a configuration thing with the task, otherwise I'll need to roll that back out and ship all the files, unfortunately.

**Comment by nathanwoulfe** [comment]
Fixed this in v1.1.0
